src/swupd/files_test.go

```diff
@@ -218,3 +219,34 @@ func TestSetHashInvalid(t *testing.T) {
 		t.Error("setHash did not fail on invalid hash")
 	}
 }
+
+func TestGetHashString(t *testing.T) {
+	// reset Hashes so we get the expected indices
+	f := File{}
+	validHash := "9bcc1718757db298fb656ae6e2ee143dde746f49fbf6805db7683cb574c36729"
+	if err := f.setHash(validHash); err != nil {
+		t.Fatal("setHash failed on valid hash")
+	}
+
+	hash := f.getHashString()
+	if hash != validHash {
+		t.Errorf("hash %v did not match expected %v", hash, validHash)
+	}
+}
+
+func TestGetFlagString(t *testing.T) {
```

Comment:
I'd expect another test with no flags set on the file to validate an error return.

Reply:
Added